src/clj/game/cards/hardware.clj

```diff
@@ -1113,5 +1113,5 @@
    "Zamba"
    {:implementation "Credit gain is automatic"
     :in-play [:memory 2]
-    :events {:expose {:effect (effect (gain :credit 1))
+    :events {:expose {:effect (effect (gain :runner :credit 1))
```

Comment:
I would have been the same issue with `(req (gain state side :credit 1))`.

Reply:
It's more that when you have to manually specify `side` you might be more likely to realise it really needs `:runner`, rather than thinking "it's an `effect`, don't need to specify a `side`"